tabpy-server/tabpy_server/app/app.py

```diff
@@ -239,6 +242,9 @@ def set_parameter(settings_key,
                               self.settings[ConfigParameters.TABPY_PWD_FILE],
                               RuntimeError)
 
+        features = self._get_features()
+        self.settings['versions'] = {'v0.4': {'features': features}}
```

Comment:
Yeah... I don't like v0.4 here. Since it's not TabPy version but rather API version it should be less cryptic. IMO just `1` is the best option for now.